servicecertsigner/install.go

```diff
@@ -0,0 +1,26 @@
+package servicecertsigner
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	servicecertsignerv1alpha1 "github.com/openshift/api/servicecertsigner/v1alpha1"
+)
+
+const (
+	GroupName = "operator.openshift.io"
```

Comment:
should be servicecertsigner.openshift.io?

Reply:
> should be servicecertsigner.openshift.io?

thanks